aw_watcher_window/main.py

```diff
@@ -54,6 +54,7 @@ def main():
     while True:
         try:
             current_window = get_current_window()
+            logger.debug(current_window)
```

Comment:
I might leave it there, if people run it with `--verbose` that's the kind of output they'd expect.